app/src/main/java/com/teamagam/gimelgimel/app/map/view/DynamicLayerActionFragment.java

```diff
@@ -17,16 +16,23 @@
 public class DynamicLayerActionFragment
     extends DrawGeometryActionFragment<DynamicLayerEditViewModel> {
 
+  private static final String DYNAMIC_LAYER_ID_KEY = "dynamic_layer_id_key";
```

Comment:
OK, OK... One second. Now I get it: The two DYNAMIC_LAYER_ID_KEY aren't really related, are they? They relate only in their meaning? You use them to pass the ID through the activity to the fragment?
Oh you... You are a computer I tell you :computer: